pep-0501.txt

```diff
@@ -32,161 +50,275 @@ the supplied user data has not been properly escaped before being passed to
 the ``os.system`` call.
 
 To address that problem (and a number of other concerns), this PEP proposes
-the complementary introduction of "i-strings" (a mnemonic for "interpolation
-template strings"), where ``f"Message with {data}"`` would produce the same
-result as ``format(i"Message with {data}")``.
+the complementary introduction of "t-strings" (a mnemonic for "template literal strings"),
+where ``f"Message with {data}"`` would produce the same
+result as ``format(t"Message with {data}")``.
 
-Some possible examples of the proposed syntax::
 
-    mycommand = sh(i"cat {filename}")
-    myquery = sql(i"SELECT {column} FROM {table};")
-    myresponse = html(i"<html><body>{response.body}</body></html>")
-    logging.debug(i"Message with {detailed} {debugging} {info}")
+While this PEP and :pep:`675` are similar in their goals, neither one competes with the other,
+and can instead be used together.
 
-
-PEP Deferral
-============
-
-This PEP is currently deferred pending further experience with :pep:`498`'s
+This PEP was previously in deferred status, pending further experience with :pep:`498`'s
 simpler approach of only supporting eager rendering without the additional
-complexity of also supporting deferred rendering.
+complexity of also supporting deferred rendering. Since then, f-strings have become very popular
+and :pep:`701` has been introduced. This PEP has been updated to reflect current knowledge of f-strings,
+and improvements from 701. It is designed to be built on top of the :pep:`701` implementation.
 
 
-Summary of differences from PEP 498
-===================================
+Summary of differences from f-strings
+=====================================
```

Comment:
I'm not sure this really fits here, after the motivation but before the actual proposal. I'd suggest making this a subsection at the end of the `Proposal` section, since it fits best there as a summary of the proposal, and the text at the end of the current proposal section leads right into it.